### ISSUE TITLE
[MAS4.2.1] [Screen Reader-Services] When the user "Open any pop-up or dialog", voiceover is reading the unnecessary information for them.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1898]
**Actual Result:**
When the user opens "Sign in with your Azure account" button, voiceover announces unnecessary information for the pop-up or dialog box.
Voiceover is reading as "Close and 6 more items" when "Sign in with your Azure account" pop-up appears.

**Expected Result:**
The voiceover should announce the information about the dialog only without providing irrelevent information.

**Note:**
Same issue observed for each and every dialog box or pop-ups throughout the application.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Sign in with your Azure account" button and activate it.
4. Observe the issue.

**User Impact:**
Users who rely on screen readers will be affected if the screen reader is reading the inappropriate info for control then the screen reader users will be misguide the info for control

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}](url)

**Attachment:**
[Unnecessary info.zip](https://github.com/microsoft/BotFramework-Emulator/files/3685952/Unnecessary.info.zip)
[Extra information.zip](https://github.com/microsoft/BotFramework-Emulator/files/3895364/Extra.information.zip)

**Comment by tonyanziano** [comment]
@Kumar2608 @awalia13 

Could we get some clarification on what exactly should be narrated in the case of the Sign in with your Azure account dialog? The term irrelevant information is unclear.

![image](https://user-images.githubusercontent.com/3452012/67596065-c3f0e480-f71c-11e9-9c59-e58f776fa861.png)

For the above dialog, is the expected result for VoiceOver to read:

> Sign in with an Azure account, dialog, Don't have an Azure Account? Sign up.

An example of the expected result would be very helpful for any issues logged in the future.

**Comment by tonyanziano** [comment]
@Kumar2608 @awalia13 friendly ping

**Comment by ShikhaMishra11** [comment]
The voiceover announces "Sign in with Azure account Insertion on word: Sign, between characters: and and 6 more items, dialog, Don't have an azure account? Sign up., link"

Here, "Insertion on word: Sign, between characters: and and 6 more items" seems irrelevant as their is only 5 interactive elements and no edit field.
I'm agree with @tonyanziano the expected, behavior should be "Sign in with an Azure account, dialog, Don't have an Azure Account? Sign up., link"

Verified On 31/10/2019
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-87002)

![regression 1898](https://user-images.githubusercontent.com/55279608/67955395-8f978f80-fc18-11e9-90be-f934abd79dcb.png)

**Comment by corinagum** [comment]
@awalia13, could we have this issue re-evaluated in the nightly? Based off the latest behavior, I think the issue might be fixed. Here is a simple example showing similar behavior of the dialog: 
 https://codepen.io/Fignewton5/pen/GRRLXEB

The dialog is read out by the narrator, but I think that's what we expect.

**Comment by ShikhaMishra11** [comment]
Verified in latest build, issue is still reproing.
[Uploading Extra information.zip…]

**Comment by corinagum** [comment]
@ShikhaMishra11 do you have that download link?

**Comment by ShikhaMishra11** [comment]
Yes we have downloaded link, we took from https://github.com/microsoft/botframework-emulator-nightlies/releases/tag/v4.6.0-88623

**Comment by corinagum** [comment]
@ShikhaMishra11 I'm hoping to get the Extra information.zip as soon as possible.

**Comment by ShikhaMishra11** [comment]
uploaded the extra information.zip folder see at the top.

**Comment by tonyanziano** [comment]
Hi @ShikhaMishra11 @awalia13 ,

I've tested this on the [lately nightly version](), and I do not experience the behavior shown in the extra information.zip attached to the original comment.

The behavior that I experience on my Mac (v10.14.6), is that the dialog opens, announces all navigable items within the dialog, the word "dialog," and then the focused item.

**Example of latest nightly:**

![dialog-emu](https://user-images.githubusercontent.com/3452012/69757730-92e84300-1112-11ea-8130-0d6148c53264.gif)


This directly aligns with the behavior [shown in the Modal Dialog example on the W3 website](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html):

![dialog-w3](https://user-images.githubusercontent.com/3452012/69757777-b3180200-1112-11ea-92da-cc786f05aac2.gif)

Since this appears to be the intended behavior, can this issue be marked as fixed? Otherwise, I'm not sure we can make VoiceOver say less without deviating from what is considered best practice.

**Comment by cwhitten** [comment]
Closing as working as intended
